docs/apis/table.md

```diff
@@ -57,6 +52,170 @@ The following dependency must be added to your project in order to use the Table
 
 Note that the Table API is currently not part of the binary distribution. See linking with it for cluster execution [here]({{ site.baseurl }}/apis/cluster_execution.html#linking-with-modules-not-contained-in-the-binary-distribution).
 
+
+Registering and Accessing Tables
```

Comment:
I put this here since this is common to both Table API and SQL. Do you mean having a "Registering Tables" section both under Table API and under SQL?